lib/ansible/modules/source_control/git.py

```diff
@@ -250,6 +256,17 @@
 from ansible.module_utils._text import to_native
 
 
+def relocate_repo(module, repo_dir, old_repo_dir, worktree_dir=None):
+    shutil.move(old_repo_dir, repo_dir)
```

Comment:
This will fail if the target dir exists

Reply:
Also, this might probably raise something like `IOError` or `OSError`, so it should also be processed with fail_json.